server/services/transactionEventsController.go

```diff
@@ -44,6 +44,104 @@ func (controller *transactionEventsController) extractEventSCDeploy(tx *transact
 	return typedEvents, nil
 }
 
+func (controller *transactionEventsController) extractEventTransferValueOnly(tx *transaction.ApiTransactionResult) ([]*eventTransferValueOnly, error) {
+	isBeforeSirius := !controller.provider.IsReleaseSiriusActive(tx.Epoch)
+	rawEvents := controller.findManyEventsByIdentifier(tx, transactionEventTransferValueOnly)
+	typedEvents := make([]*eventTransferValueOnly, 0)
+
+	for _, event := range rawEvents {
+		if isBeforeSirius {
+			typedEvent, err := controller.decideEffectiveEventTransferValueOnlyBeforeSirius(event)
```

Comment:
Logic for "before Sirius" was kept as it was (even though it has only been validated for non-contract addresses).